docs/management/cases/manage-cases.asciidoc

```diff
@@ -54,6 +54,34 @@ go back to your case.
 After a visualization has been added to a case, you can modify or interact with
 it by clicking the *Open Visualization* option in the comment menu.
 
+[[add-case-notifications]]
+=== Add email notifications
+
+You can configure email notifications that occur when users are assigned to
+cases.
+
+For hosted {kib} on {ess}:
+
+. Add the email addresses to the monitoring email allowlist. Follow the steps in
+{cloud}/ec-watcher.html#ec-watcher-allowlist[Send alerts by email].
++
+--
+You do not need to take any more steps to configure an email connector or update
+{kib} user settings, since the preconfigured Elastic-Cloud-SMTP connector is
+used by default.
+--
+
+For self-managed {kib}:
+
+. Create an <<email-action-type,email connector>>. For examples, refer to
```

Comment:
Only pre-configured connectors are supported at the moment. I think we should emphasize that in the docs.

Reply:
Thanks for the clarification!  I've added a note via 64cf8372ee6fa5e86e153caa90dff17ce4f4e6a6